README.MD

```diff
@@ -0,0 +1,11 @@
+# Seedqueue
+## Description
+SeedQueue is a mod that is meant to replace multi-instancing. Instead of having multiple Minecrafts generating worlds open at the same time, it does it all in just one Minecraft instance. This greatly improves performance, especially for lower end hardware, and is also aimed to make speedrunning more accessible.
```

Comment:
Request: Please add information about the wall screen, you can take inspiration from the SeedQueue 1.0 release message.

Reply:
> Request: Please add information about the wall screen, you can take inspiration from the SeedQueue 1.0 release message.

@KingContaria i'll get back to this readme in at night, right now i have some work